### PR TITLE
feat: seoul recap blog post

### PR DESCRIPTION
**Pull request by leohhhn**
## Description

Adds the Seoul recap blog post.

Written by @michelleellen

### CODE REVIEW

**Comment by michelleellen** [comment]
This draft needs some work given the Seoul triage repo is closed, which changes the paragraph.

**Review by michelleellen — APPROVED** [review]

File: posts/2024-11-28_seoul-retreat/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+---
+
+# Gnomes spotted in Seoul: Onbloc x Core Engineering Collaboration 
+
+gno.land’s core engineering team has just wrapped up an intense and highly 
+productive two-week coding sprint with our core contributor, 
+[Onbloc](https://onbloc.xyz/). Hosted in Onbloc's offices in the heart of Seoul,
+this onsite collaboration stands as one of our most successful development cycles
+yet, setting new records for pull requests merged and advancing critical priorities
+for our upcoming mainnet launch.
```
Comment (zivkovicmilos) [inline posts/2024-11-28_seoul-retreat/README.md:15]: I would throw in: `, especially for the GnoVM`.

File: posts/2024-11-28_seoul-retreat/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+for iterative feature releases, ensuring adaptability and a user-centric experience 
+throughout the process. Key features include:
+- Design Overhaul: Creating a more user-friendly UI with a color palette 
+matching gno.land's true spirit.
+- UX Redesign: Establishing `gnoweb` as the go-to realm web frontend.
+- Code Refactoring: Enhancing security and maintainability.
+- Navigation Improvements: Simplifying user navigation and introducing a query 
+system for direct linking to specific `gnoweb` content or commands.
+
+Finally, the team collaborated closely with Onbloc’s DevOps experts to streamline
```
Comment (zivkovicmilos) [inline posts/2024-11-28_seoul-retreat/README.md:37]: We can add more things here, `gnoweb` was just one project:

https://hackmd.io/@JFGdLdZISka23j6-v8UYbA/H17Awcf7Jg/edit


We also helped Onbloc identify pain points in current Gnoswap, Adena and Gnoscan implementations, through a workshop with both teams

File: posts/2024-11-28_seoul-retreat/src/thumbs/meetup.jpg
Comment (zivkovicmilos) [inline posts/2024-11-28_seoul-retreat/src/thumbs/meetup.jpg:1]: I still don't understand why these blog post images are compressed like we're showing them in 2003

File: posts/2024-11-28_seoul-retreat/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+developers attended the event, eager to learn about gno.land’s vision and explore 
+how they could contribute. The enthusiasm from local developers and the 
+collaboration with Onbloc highlight the growing global momentum behind gno.land. 
+To further support regional growth, we are prioritizing multilingual content,
+starting with Korean. In collaboration with Onbloc, we created Korean video 
+content for the community - coming soon! 
+
+[![office](https://gnolang.github.io/blog/2024-11-28_seoul-retreat/src/thumbs/meetup.jpg)](https://gnolang.github.io/blog/2024-11-28_seoul-retreat/src/meetup.jpg)
+
+## A Celebration of Culture and Collaboration
```
Comment (zivkovicmilos) [inline posts/2024-11-28_seoul-retreat/README.md:56]: This section is great 👌

**Review by zivkovicmilos — APPROVED** [review]
One of the best recaps so far, @michelleellen 👏 

I've added a few comments, but otherwise good to go 💯

File: posts/2024-11-28_seoul-retreat/src/thumbs/meetup.jpg
Comment (michelleellen) [reply posts/2024-11-28_seoul-retreat/src/thumbs/meetup.jpg:1]: I can't help you here :)

**Review by michelleellen — APPROVED** [review]

**Review by michelleellen — APPROVED** [review]
I made the requested changes to the blog post

File: posts/2024-11-28_seoul-retreat/README.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+[![office](https://gnolang.github.io/blog/2024-11-28_seoul-retreat/src/thumbs/office.jpg)](https://gnolang.github.io/blog/2024-11-28_seoul-retreat/src/office.jpg)
+
+## Starting with a Bang: test5 and engineering developments
+
+The collaboration kicked off with the launch of test5. This milestone set the tone for two weeks of unparalleled productivity. During this sprint, the teams identified areas of collaboration like a feedback workshop on Gnoscan, Gnoswap, and Adena Wallet, and topics such as gas, realm upgradeability, gnoweb, tx-indexer, infrastructure, and ABCI queries. The benchmarking and gas cost estimations is a priority as the team aims to optimize transaction efficiency for both users and developers as we work our way towards mainnet. Realm development will advance significantly in the next few months, highlighting the need to enhance an existing realm's functionality and capabilities without creating a new one. To address this, a realm upgradeable pattern example is necessary to illustrate possible solutions to this challenge. This feature is particularly important for applications like Gnoswap, and was a key focus during the recent two-week sprint.
```
Comment (leohhhn) [inline posts/2024-11-28_seoul-retreat/README.md:21]:
```suggestion
The collaboration kicked off with the launch of test5. This milestone set the tone for two weeks of unparalleled productivity. During this sprint, the teams identified areas of collaboration including a feedback workshop on Gnoscan, Gnoswap, and the Adena Wallet, and topics such as gas, realm upgradeability, `gnoweb`, `tx-indexer`, infrastructure, and ABCI queries. The benchmarking and gas cost estimations are a priority as the team aims to optimize transaction efficiency for both users and developers. Realm development will advance significantly in the next few months, highlighting the need to enhance an existing realm's functionality and capabilities without creating a new one. To address this, a realm upgradeable pattern example is necessary to illustrate possible solutions to this challenge. This feature is particularly important for applications like Gnoswap, and was a key focus during the recent two-week sprint.
```

**Pull request merged**